core/adapters/random_test.go

```diff
@@ -0,0 +1,44 @@
+package adapters_test
+
+import (
+	"math/big"
+	"testing"
+
+	"github.com/smartcontractkit/chainlink/core/adapters"
+	"github.com/smartcontractkit/chainlink/core/store/models"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestRandom_Perform(t *testing.T) {
+	tests := []struct {
+		name    string
+		errored bool
+	}{
+		{"resulting string parses to a big.Int 1", false},
+		{"resulting string parses to a big.Int 2", false},
+		{"resulting string parses to a big.Int 3", false},
+		{"resulting string parses to a big.Int 4", false},
+		{"resulting string parses to a big.Int 5", false},
+	}
+
+	for _, tt := range tests {
+		test := tt
+		t.Run(test.name, func(t *testing.T) {
+			t.Parallel()
```

Comment:
We've made it a standard to only place `t.Parallel` at the top level and not in subtests. Let's place this on line 13.